src/Illuminate/Routing/Route.php

```diff
@@ -485,6 +485,10 @@ public function signatureParameters($subClass = null)
      */
     public function bindingFieldFor($parameter)
     {
+        if (is_int($parameter)) {
```

Comment:
It just a suggestion. 
```
    $bindingFields = $this->bindingFields;
    if (is_int($parameter)) {
        $bindingFields = array_values($bindingFields);
    }
    
    return $bindingFields[$parameter] ?? null;
```

Or

```
    $bindingFields = $this->bindingFields;
    
	return (is_int($parameter) ? array_values($bindingFields)[$parameter] : bindingFields[$parameter]) ?? null;
```

Reply:
I did it first like 

```php
    $bindingFields = is_int($parameter) ? array_values($this->bindingFields) : $this->bindingFields;

        return $bindingFields[$parameter] ?? null;
```

but I wanted the code to be more readable